keps/sig-auth/3257-trust-anchor-sets/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Trust Anchor Sets
+kep-number: 3257
+authors:
+  - "@ahmedtd"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
```

Comment:
I'd expect to see:
```suggestion
 - sig-auth
   - sig-security
```

Reply:
Done

Reply:
Certificates is a sig-auth subproject. sig-security has so far not participated in this or other work in the certificate subproject.

Reply:
Un-done :P